unity-client/Assets/Scripts/MainScripts/DCL/Models/Quests/QuestModel.cs

```diff
@@ -11,6 +12,15 @@ public static class Status
         public static string COMPLETED = "completed";
         public static string FAILED = "failed";
     }
+
+    public static class RewardStatus
+    {
+        public static string NOT_GIVEN = "not_given";
```

Comment:
all these static string vars could be CONSTs if we don't change them in runtime.

Reply:
I have no idea why I didnt make them `consts`, I will change them!